lib/utils.js

```diff
@@ -313,7 +314,7 @@ function removeUndefined(query, handleUndefined) {
           this.update(null);
           break;
         case 'throw':
-          throw new Error('Unexpected `undefined` in query');
+          throw new Error(g.t('Unexpected `undefined` in query'));
```

Comment:
sorry I think it should be `{{`undefined`}}` instead of `{{undefined}}`. And if `g.f` skips it successfully, the error message should not change right?